src/common/error_log.cpp

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include "common/error_log.hpp"
+
+#include <fcntl.h>
+#include <fmt/format.h>
+
+#include <phosphor-logging/elog.hpp>
+
+namespace hw_isolation
+{
+namespace error_log
+{
+
+using namespace phosphor::logging;
+
+FFDCFile::FFDCFile(const FFDCFormat& format, const FFDCSubType& subType,
+                   const FFDCVersion& version, const std::string& data) :
+    _format(format),
+    _subType(subType), _version(version),
+    _fileName((std::string("/tmp/hwIsolation") +
+               CreateIface::convertFFDCFormatToString(_format) +
+               std::string(".XXXXXX"))),
```

Comment:
you could aswell place it in the constructor for readability/maintainability, just a suggestion no chg required

Reply:
Ack